From 2579d9b982c7232f9354bcca5262e26a84c38799 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Fri, 2 Nov 2018 17:40:32 +0100
Subject: [PATCH] nvmem: make CONFIG_NVMEM tristate again

Only build it in when OF_NET is selected and make it possible to build
it as module otherwise.

Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 drivers/nvmem/Kconfig | 2 +-
 drivers/of/Kconfig    | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/nvmem/Kconfig
+++ b/drivers/nvmem/Kconfig
@@ -1,5 +1,5 @@
 menuconfig NVMEM
-	bool "NVMEM Support"
+	tristate "NVMEM Support"
 	help
 	  Support for NVMEM(Non Volatile Memory) devices like EEPROM, EFUSES...
 
--- a/drivers/of/Kconfig
+++ b/drivers/of/Kconfig
@@ -71,6 +71,7 @@ config OF_IRQ
 
 config OF_NET
 	depends on NETDEVICES
+	select NVMEM
 	def_bool y
 
 config OF_MDIO
